src/ImageRunModal.jsx

```diff
@@ -119,6 +119,7 @@ const Volume = ({ id, item, onChange, idx, removeitem, additem }) =>
                        placeholder={_("Container path")}
                        value={item.containerPath || ''}
                        onChange={e => onChange(idx, 'containerPath', e.target.value)} />
+                <div />
```

Comment:
Was that on purpose, or some kind of leftover noise?

Reply:
On purpose. There are 4 elements we want to show. I want to have two in one row and two in the next. But the form puts 3 items on one line and then wraps to another, so putting one empty div makes things wrap as expected.